apps/cli/src/diagnostic/DebugLlmApi.ts

```diff
@@ -7,7 +7,8 @@ export class DebugLlmApi implements LlmApi {
   constructor(
     public debugLog: DebugLog,
     public llm: LlmApi,
-  ) { }
+    public chat: Chat
```

Comment:
The chat shouldn't be passed in on construction, instead we want to capture the message-log whenever `getResponse` is called, so that we can save it out to the debug-log before it's sent to the LLM.